guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -9,24 +9,23 @@ To create a file type repository in a directory on the base system where {Projec
 
 .Prerequisites
 ifdef::katello[]
-* You have a server running {EL} 7 or 8 registered to your {Project}.
+* You have a server running {EL} 8 registered to your {Project}.
 endif::[]
 ifdef::satellite[]
-* You have a server running {EL} 7 or 8 registered to your {Project} or the Red{nbsp}Hat CDN.
+* You have a server running {EL} 8 registered to your {Project} or the Red{nbsp}Hat CDN.
 * Your server has an entitlement to the {RHELServer} and {ProjectName} Utils repositories.
 endif::[]
 ifdef::orcharhino[]
 * You have a server running {EL} registered to your {Project}.
 endif::[]
 * You have installed an HTTP server.
 ifndef::orcharhino[]
-For more information about configuring a web server, see {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/setting-apache-http-server_deploying-different-types-of-servers[Setting up the Apache HTTP web server] in {RHEL}{nbsp}8 _Deploying different types of servers_ or {RHELDocsBaseURL}7/html/system_administrators_guide/ch-web_servers#s1-The_Apache_HTTP_Server[The Apache HTTP Server] in the _{RHEL}{nbsp}7 System Administrator's Guide_.
+For more information about configuring a web server, see {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/setting-apache-http-server_deploying-different-types-of-servers[Setting up the Apache HTTP web server] in {RHEL}{nbsp}8 _Deploying different types of servers_.
 endif::[]
 
 .Procedure
 ifdef::satellite[]
 . On your server, ensure that the right repositories are enabled.
```

Comment:
```suggestion
. On your server, ensure that the right repositories are enabled:
```

alternative, but unrelated: ". On your server, enable the required repositories:"